test_scripts/UpdateVideoStreamingCapabilities/002_OnAppCapabilityUpdated_from_mobile_to_HMI_capability_parameter_is_missed.lua

```diff
@@ -0,0 +1,42 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description:
+-- Processing OnAppCapabilityUpdated notification from mobile to HMI
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. App1 with NAVIGATION appHMIType is registered
+-- 3. OnAppCapabilityUpdated notification is alowed by policy for App1
+--
+-- Sequence:
+-- 1. App1 sends OnAppCapabilityUpdated for VIDEO_STREAMING capability type without 
+--    videoStreamingCapability parameter
+-- 2. SDL sends OnAppCapabilityUpdated notification to the HMI
+
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+local appCapability = {
+  appCapability = {
+    appCapabilityType = "VIDEO_STREAMING",
+    -- videoStreamingCapability is missed
+  }
+}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Prepare preloaded policy table", common.preparePreloadedPT)
```

Comment:
This is the precondition for the most part of scripts so propose to move to function `precondition` in common file